gensrc/thrift/BackendService.thrift

```diff
@@ -265,8 +265,8 @@ struct TWorkloadGroupInfo {
   9: optional i32 scan_thread_num
   10: optional i32 max_remote_scan_thread_num
   11: optional i32 min_remote_scan_thread_num
-  12: optional i32 spill_threshold_low_watermark
-  13: optional i32 spill_threshold_high_watermark
+  12: optional i32 memory_low_watermark
+  13: optional i32 memory_high_watermark
```

Comment:
Did a upgrade test here? Such as  a old FE and a new BE with this PR, what could happen?

Reply:
Test is done and it's OK to rename a field.